src/elements/content-sidebar/stories/tests/BoxAISidebar-visual.stories.tsx

```diff
@@ -1,19 +1,49 @@
-import { expect, userEvent, within } from '@storybook/test';
+import {expect, fn, screen, userEvent, waitFor, within} from '@storybook/test';
 import { type StoryObj } from '@storybook/react';
+import {AgentType} from '@box/box-ai-agent-selector';
 import ContentSidebar from '../../ContentSidebar';
 import BoxAISidebar from '../../BoxAISidebar';
 
 const mockFeatures = {
     'boxai.sidebar.enabled': true,
 };
 
+export const mockAgents: AgentType[] = [
```

Comment:
when you export from the stories, it gets added to the storybook. if you need to share this mockAgents, move it to a constants file or somewhere else